modules/steam/hm.nix

```
@@ -0,0 +1,60 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.stylix.targets.steam;
+
+  adwaitaCustomCss = config.lib.stylix.colors {
+    template = ./custom.mustache;
+    extension = "css";
+  };
+
+  adwaitaTheme = pkgs.stdenv.mkDerivation (_self: {
+    name = "Adwaita-for-Steam";
+    version = "3.1";
+
+    src = config.stylix.inputs.adwaita-for-steam;
+
+    buildInputs = with pkgs; [ python3 ];
+
+    buildPhase = ''
+      mkdir --parents $out/steamui/css
+      touch $out/steamui/css/library.css
+      cp ${adwaitaCustomCss} ./custom/custom.css
+      python3 install.py --custom-css --target $out
```

The `pkgs.python3` package provides `/bin/python` which symlinks to the same binary as `/bin/python3`:

```suggestion
      python install.py --custom-css --target $out
```

Removing the "3" looks neater. But I have no strong feelings about this.